### ISSUE TITLE
Cannot edit files (Change Date or Stack) belonging to Partner's Account

### ISSUE THREAD

**Issue by ohare93** [immich-app/immich#13839]
### The bug

In the main view I cannot edit files. I have a lot I wish to Change the Date of to the date of the event so I multi-select them and hit the button but nothing happens. Upon investigating in the Network tab I can see it's doing a PUT towards `https://photos.mysite.com/api/assets` with the payload containing an empty list of ids 🤔 

```json
{"ids":[],"dateTimeOriginal":"2024-08-11T00:00:00.000+02:00"}
```

When multi-selecting a different set of images and selecting Stack nothing at all happens, no network events either. No changes upon reloading. Same thing on web and mobile.

### The OS that Immich Server is running on

Unraid 6.12.6

### Version of Immich Server

v1.119.1

### Version of Immich Mobile App

1.119.0 build.164

### Platform with the issue

- [ ] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
#
# WARNING: Make sure to use the docker-compose.yml of the current release:
#
# https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml
#
# The compose file on main may not be compatible with the latest release.
#

name: immich-ohare

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    # extends:
    #   file: hwaccel.transcoding.yml
    #   service: cpu # set to one of [nvenc, quicksync, rkmpp, vaapi, vaapi-wsl] for accelerated transcoding
    volumes:
      # Do not edit the next line. If you want to change the media storage location on your system, edit the value of UPLOAD_LOCATION in the .env file
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
      - /mnt/user/Pictures:/mnt/media/Pictures
    env_file:
      - .env
    # ports:
    #   - 2283:3001
    depends_on:
      - redis
      - database
    restart: always
    healthcheck:
      disable: false
    network_mode: proxynet
    labels:
      - traefik.enable=true
      - traefik.http.routers.immich-ohare.entryPoints=https
      - traefik.http.routers.immich-ohare.rule=Host(`photos.mysite.com`)
      - traefik.http.routers.immich-ohare.middlewares=auth@file
      # - traefik.docker.network=traefik

  immich-machine-learning:
    container_name: immich_machine_learning
    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
    # Example tag: ${IMMICH_VERSION:-release}-cuda
    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/ml-hardware-acceleration
    #   file: hwaccel.ml.yml
    #   service: cpu # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
      - model-cache:/cache
    env_file:
      - .env
    restart: always
    healthcheck:
      disable: false
    network_mode: proxynet
    # networks:
    #   - immich-ohare

  redis:
    container_name: immich_redis
    image: docker.io/redis:6.2-alpine@sha256:2d1463258f2764328496376f5d965f20c6a67f66ea2b06dc42af351f75248792
    healthcheck:
      test: redis-cli ping || exit 1
    restart: always
    network_mode: proxynet
    # networks:
    #   - immich-ohare

  database:
    container_name: immich_postgres
    image: docker.io/tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
      POSTGRES_INITDB_ARGS: --data-checksums
    volumes:
      # Do not edit the next line. If you want to change the database storage location on your system, edit the value of DB_DATA_LOCATION in the .env file
      - ${DB_DATA_LOCATION}:/var/lib/postgresql/data
    # ports:
    #   - ${DB_PORT}:5432
    healthcheck:
      test: pg_isready --dbname='${DB_DATABASE_NAME}' --username='${DB_USERNAME}' ||
        exit 1; Chksum="$$(psql --dbname='${DB_DATABASE_NAME}'
        --username='${DB_USERNAME}' --tuples-only --no-align --command='SELECT
        COALESCE(SUM(checksum_failures), 0) FROM pg_stat_database')"; echo
        "checksum failure count is $$Chksum"; [ "$$Chksum" = '0' ] || exit 1
      interval: 5m
      start_interval: 30s
      start_period: 5m
    command:
      - postgres
      - -c
      - shared_preload_libraries=vectors.so
      - -c
      - search_path="$$user", public, vectors
      - -c
      - logging_collector=on
      - -c
      - max_wal_size=2GB
      - -c
      - shared_buffers=512MB
      - -c
      - wal_compression=on
    restart: always
    network_mode: proxynet

volumes:
  model-cache: null
networks: {}
```

### Your .env content

```Shell
# You can find documentation for all the supported env variables at https://immich.app/docs/install/environment-variables

# The location where your uploaded files are stored
UPLOAD_LOCATION=/mnt/user/MiscAppData/immich-ohare/files
# The location where your database files are stored
DB_DATA_LOCATION=/mnt/user/MiscAppData/immich-ohare/db

# To set a timezone, uncomment the next line and change Etc/UTC to a TZ identifier from this list: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List
TZ=Europe/Colenhagen

# The Immich version to use. You can pin this to a specific version like "v1.71.0"
IMMICH_VERSION=v1.119.1

# Connection secret for postgres. You should change it to a random password
# Please use only the characters `A-Za-z0-9`, without special characters or spaces
DB_PASSWORD=Password
# DB_PORT=5433
DB_HOSTNAME=immich_postgres

# The values below this line do not need to be changed
###################################################################################
DB_USERNAME=postgres
DB_DATABASE_NAME=immich
```

### Reproduction steps

1. Multi-select something in the main screen
2. Select Date Change or Stack

Nothing happens

### Relevant log output

```shell
I don't see any logs occurring when I hit these buttons. When looking in Dozzle, at least.
```

**Comment by ohare93** [comment]
I have discovered the issue: all the photos I wish to edit (either their metadata or to stack them) are my wife's. They were uploaded by her on her own account in Immich. I enabled Partner Share for us both so we can see each other's images in the timeline, yet apparently I do not have edit permissions...?

Logging into my wife account and trying all of the above works. I can change the dates, I can stack, all with no issue. Knowing what I do now I logged back into my account and tried to Stack photos that are my own, and it worked. Same with the Date change.

Very bad UX to not have any sort of feedback to the user that the action they try and take will not do anything, that could easily be improved. I presume it's on purpose, and that's why I had an empty list in the payload above. So the app does know some images will not be impacted, and thus it could be a time to inform the user with a popup.

All that aside, why can't I edit my wife's images? We'd like that ability. Does Partner Sharing not allow for this?
